packages/synthetix-cli/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "@synthetixio/synthetix-cli",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": ""
+  },
+  "author": "",
+  "license": "MIT",
+  "devDependencies": {
+    "@nomiclabs/hardhat-ethers": "^2.0.2",
+    "ethers": "^5.5.1",
```

Comment:
ethers is included in @nomiclabs/hardhat-ethers

Reply:
I think it's alright, since he's following the pattern used in the deployer package. If you do apply Leo's suggestion, I think it should be on both plugins.